spec/hardware/system.fmf

```diff
@@ -1,17 +1,37 @@
-summary:
-    Select machine with given system attributes
+summary: |
+    Select or provision a guest with given system properties.
+
+description: |
+    .. code-block::
+
+       system:
+           # Number or string, an ID of the device vendor.
+           vendor: 1234|"> 1234"
+
+           # String, a name of the device vendor.
+           vendor-name: "A Well-Known Device Manufacturer"
+
+           # Integer or string, system model ID.
+           model: 1234|"> 1234"
+
+           # String, system model name.
+           model-name: ProLiant DL385 Gen10
+
+           # Integer or string, required number of NUMA nodes.
+           numa-nodes: 2|">= 2"
+
+    .. versionchanged:: 1.29
+       Fixed the naming scheme of ``model`` and ``vendor`` keys.
+       See https://github.com/teemtee/tmt/pull/2431.
```

Comment:
This is a draft, not implemented, is the change that important that it deserves the pull request link?

Reply:
Fair enough, dropped.